src/net/tcpsock.go

```diff
@@ -88,6 +88,11 @@ func ResolveTCPAddr(network, address string) (*TCPAddr, error) {
 		return nil, fmt.Errorf("Network '%s' not supported", network)
 	}
 
+	switch address {
+	case ":http":
+		address = ":80"
```

Comment:
Does native (upstream) Go default to port 80 too?

Reply:
Yes, here's the doc for Addr in http.Server:

```
type Server struct {
	// Addr optionally specifies the TCP address for the server to listen on,
	// in the form "host:port". If empty, ":http" (port 80) is used.
	// The service names are defined in RFC 6335 and assigned by IANA.
	// See net.Dial for details of the address format.
	Addr string
```